test/memory.cpp

```diff
@@ -0,0 +1,17 @@
+#include "gtest/gtest.h"
+
+#include "../src/platform/memory.h"
+
+TEST(Test_GetFreeRAMSize, Max) {
+    bool status;
+    unsigned long long testVar1, testVar2;
+    void** buffer;
+
+    testVar1 = GetFreeRAMSize();
+    status = allocatePool(30, buffer);
+    testVar2 = GetFreeRAMSize();
+
+    EXPECT_EQ(testVar1, testVar2 - 30);
+}
+
+
```

Comment:
Have you tried to run this code? I would need to be included into the test project. But, the test projects runs in a MS Windows environment without UEFI functions, so it will fail. Having unit tests only makes sense if you provide a Windows implementation of the functions used in your function, which is not required here. I recommend to remove this file and do tests in VM and test-net only..